Main/Workshop3/Item.cpp

```diff
@@ -0,0 +1,15 @@
+#include "Item.h"
+
+#include <utility>
+
+Item::Item(std::string name, const uint_fast8_t bonus) : name(std::move(name)), bonus(bonus) {}
+
+Item::~Item() = default;
+
+std::string Item::getName() const {
+    return name;
+}
+
+uint_fast8_t Item::getBonus() const {
```

Comment:
this one is ok but they are all the same after all. And what about Item per specific monster class? Or affected by user level? Or by type, like `head`, `armor`, `shoes` etc. Or ... or ... or...